src/documentation/pages/components/header/examples/index.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+  div
+    h3 Examples
+    p.-text 
+      | To use headers, apply the class <code>chi-header</code> to <code>&lt;header&gt;</code>
+      | and wrap all content in <code>chi-header__content</code>. Use <code>chi-header__start</code>
+      | to position elements at the start of the header and <code>chi-header__end</code> to
+      | position elements at the end of the header.
+    h3 Responsiveness
+    p.-text
+      | By default, all content stored in headers will render in drawer
+      | components when the viewport is too small to accommodate.
+
+    div
+      <Base />
```

Comment:
Compare with the documentation (https://assets.ctl.io/chi/latest/components/header/) the different examples for each theme: Lumen, CenturyLink and portal.

CenturyLink has 9 examples, Portal has 2 and Lumen has 12 examples, so what you need to do is to split them with these divs:

    div(v-if="$store.state.themes.theme === 'lumen'")
    div(v-if="$store.state.themes.theme === 'portal'")
    div(v-if="$store.state.themes.theme === 'centurylink'")

Reply:
Added example components for all the themes as required.